### PR TITLE
Feature/register

### PR DESCRIPTION
**Pull request by caesargusti**
models/accounts
api/customers/customer

### CODE REVIEW

File: helpers/helpers.go [hunk 1]
```diff
@@ -27,7 +27,8 @@ func LoadConfig(file string) (models.Config, error) {
 		return models.Config{}, err
 	}
 	return cfg, err
```
Comment (sekarminati) [inline helpers/helpers.go]: Please change this to "return"

File: models/accounts.go [hunk 2]
```diff
@@ -21,3 +21,13 @@ func GetMainAccount(db *sql.DB, accNum string) (Accounts, error) {
...
 }
+
+func AddAccountsWhileRegister(db *sql.DB, AccNum string) error {
+	Create := time.Now()
+	Ammount := 0
+	_, err := db.Exec("INSERT into accounts(account_num, account_balance, created_at) values ($1, $2, $3)", AccNum,
+			Ammount,
+			Create,
+		)
+		return err
```
Comment (sekarminati) [inline models/accounts.go:36]: Please change this to "return"

**Review by sekarminati — CHANGES_REQUESTED** [review]

**Review by david1312 — APPROVED** [review]
good to go

File: api/customers/customers.go [hunk 3]
```diff
@@ -25,14 +42,42 @@ func NewCustomerHandler(jwt *tokens.JWT, db *sql.DB) *CustomerHandler {
...
+	res, err := json.Marshal(result)
+	if err != nil {
+		helpers.HTTPError(w, http.StatusBadRequest, err.Error())
+		return
+	}
+
+	fmt.Fprintln(w, string(res))
 }
 
+var emailRegex = regexp.MustCompile("^[a-zA-Z0-9.!#$%&'*+\\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$")
```
Comment (yulyulyharuka) [inline api/customers/customers.go:79]: you can move emailRegex into inside isEmailValid function because u dont need this globally

**Review by yulyulyharuka — CHANGES_REQUESTED** [review]

File: api/customers/customers.go [hunk 4]
```diff
@@ -70,20 +77,56 @@ func (ch *CustomerHandler) Create(w http.ResponseWriter, r *http.Request){
...
+		Email: cus.CustEmail,
+	}
+
+	err = json.NewEncoder(w).Encode(dataemail)
+	if err != nil {
+		helpers.HTTPError(w, http.StatusBadRequest, "Unable to Encode response")
+		return
+	}
+
+	log.Println(string(body))
```
Comment (yulyulyharuka) [inline api/customers/customers.go]: check if the body is error or not, if the api returning error then handle it

**Review by yulyulyharuka — CHANGES_REQUESTED** [review]

**Review by vicimilenia — APPROVED** [review]

**Pull request merged**